torch_xla_py/xla_model.py

```diff
@@ -83,6 +83,14 @@ def get_xla_supported_devices(devkind=None, max_devices=None):
       return kind_devices[:max_devices] if max_devices else kind_devices
 
 
+def xrt_world_size():
+  return int(os.getenv('XRT_NUM_WORKERS', 1))
```

Comment:
Actually, I have a second thought about this.
We should be (in the `xla_dist.py` case) exporting explicit variables about "ordinal" and "world size".
Maybe eve using the same of pytorch?

Reason is, with the new multiprocessing, there will be one shard per core, for the concept of worker becomes a little loose.
In that case, a "worker" is meant as a process-group handling one TPU device, and each of the "forks" will have a device assigned.

Reply:
Ok, let me see if we're on the same page.

Let's say we have 1:1 setup between Compute VM and TPU VM, and the TPU is a `v3-32`. Then in the multiprocessing world, we'd have 4 workers (aka. process groups) but 32 separate processes/shards. So in that case we would have:
* `XRT_LOCAL_WORKER=c_tpu_worker:0...3`
* `XRT_SHARD_ORDINAL=0...31`
* `XRT_SHARD_WORLD_SIZE=32`

Is that correct?

Reply:
Also, during the meantime while we don't have multiprocessing, I'll set `XRT_SHARD_ORDINAL=0...3` and `XRT_SHARD_WORLD_SIZE=4` since 1 shard = 1 worker = 1 process until we have multiprocessing.

Reply:
For now, you only set XRT_SHARD_ORDINAL=0..3 and XRT_SHARD_WORLD_SIZE=4.
The multiprocessing will have to rewrite those.
We will have to support (at least for a while) bot the multi-thread and multi-processing interfaces anyway.

Reply:
Done.